website/project/tasks.py

```diff
@@ -72,8 +74,8 @@ def _async_update_node_share(self, node_id):
     AbstractNode = apps.get_model('osf.AbstractNode')
     node = AbstractNode.load(node_id)
 
-    data = serialize_share_node_data(node)
-    resp = send_share_node_data(data)
+    data = serialize_share_data(node)
+    resp = send_share_json(data)
```

Comment:
To address the instead of writing any entirely new tests. I simply removed some of  the mocking from `TestOnNodeUpdate` this caused the above error to manifest, so I simply fixed it so those tests now pass properly and cover this situation.